backend/src/model/entity/plants_impl.rs

```diff
@@ -36,6 +36,13 @@ impl Plants {
         page_parameters: PageParameters,
         conn: &mut AsyncPgConnection,
     ) -> QueryResult<Page<PlantsSummaryDto>> {
+        // Set higher search sensitivity so that the user gets more search results than with default
+        // settings (0.3).
+        let set_similarity_threshold =
+            diesel::sql_query("SET LOCAL pg_trgm.similarity_threshold=0.1");
+        debug!("{}", debug_query::<Pg, _>(&set_similarity_threshold));
+        let _ = set_similarity_threshold.execute(conn).await;
+
```

Comment:
Is it possible to somehow avoid having to make two calls to PostgreSQL? I just had a quick look and it seems difficult do avoid this, but maybe you can think of a different solution.

Reply:
The only alternative I see is to set the value of this variable globally using an ALTER DATABASE command (e.g. as part of a migration script).

Reply:
Then it should be fine like it is currently. I don't think we should invest to much time here.